aiidalab_widgets_base/viewers.py

```diff
@@ -15,12 +18,23 @@ def viewer(obj, downloadable=True, **kwargs):
     :type downloadable: bool
 
     Returns the object itself if the viewer wasn't found."""
+    import inspect
+    import warnings
+    from aiida.orm import Node
+
+    if not (inspect.isclass(type(obj)) and issubclass(type(obj), Node)):  # only working with AiiDA nodes
```

Comment:
```suggestion
    if not isinstance(obj, Node)):  # only working with AiiDA nodes
```

Reply:
this doesn't work with python3.7

Reply:
No, you are right. I confused myself. `issubclass` in Python 3.7 expects a class, not an object. `isinstance` works fine.